app/views/career_site/positions/show.html.slim

```diff
@@ -47,13 +47,15 @@
           form,
           method: :full_name,
           required: true,
+          size: :medium,
           placeholder: "Full name" \
         )
       = render TextInputComponent.new( \
           form,
           method: :email,
           required: true,
           type: :email,
+          size: :medium,
```

Comment:
not sure that input takes medium size, on lookbook it has different padding
we took styles from `form-control`
![image](https://github.com/user-attachments/assets/9d90e408-bae3-447f-9ee5-139f9fcdd0a9)